### DISCUSSION TITLE
Blockscout sent large of eth_call requests?

### DISCUSSION THREAD

**Discussion by Eoous** [blockscout/blockscout#9794] [Q&A]
Deployed my own blockscout, and foud large of eth_call requests?

Why blockscout would send these requests?

**Comment by sl1depengwyn** [comment]
Blockscout uses it to call contract functions, it used for read contract view, for fetching revert reason and for indexing nfts

**Reply by Eoous** [reply]
For indexing nfts, why not using log event?

**Reply by sl1depengwyn** [reply]
There is no URI in logs
